src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <Display result="something display" />
+      <ButtonPanel />
+    </div>
```

Comment:
- [ ] **(OPTIONAL)** You can use [React Fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax) here if you don't want this `div`.

Reply:
@muriloroque thank you a lot for the review